contrib/install/install.sh

```diff
@@ -0,0 +1,210 @@
+#!/bin/bash
+#
+#
+#  Copyright 2016 CUBRID Corporation
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+function get_cubrid_dir() {
+  cubrid_str=/CUBRID/
+  cubrid_str_len=${#cubrid_str}
+  current_dir="`pwd`"
+
+  find_dir=$(echo ${current_dir} | grep ${cubrid_str})
+
+  if [ -z "$find_dir" ];then
+    echo ""
+  else
+    temp_dir=${current_dir%%$cubrid_str*}
+    temp_dir_len=${#temp_dir}
+    cubrid_dir_len=`expr $temp_dir_len + $cubrid_str_len`
+    cubrid_dir=$(echo $current_dir | cut -c1-$cubrid_dir_len)
+    echo "$cubrid_dir"
+  fi
+}
+
+
+cubrid_home=$(get_cubrid_dir)
+
+if [ -z "$cubrid_home" ];then
+  cubrid_home="`pwd`"
+fi
+
+echo "Is CUBRID installed in the directory below? [Yn]:"
+echo "\"$cubrid_home\""
```

Comment:
Suggest:
echo "Is the CUBRID installed in \"$cubrid_home\" ? [Yn]:"

Reply:
If the path is long, readability is not good, so it is expressed in two lines.
let us know your opinion.

Reply:
It is the style of the cubrid install shell script.
Personally, I prefer single line even it is too long, but it doesn't matter.
Do as you like.

Reply:
Thanks for an opinion.
I agree with your opinion.

Reply:
Because the last line is not a question, but a proposed CUBRID directory, I have to read previous line to answer. It is little bit confused. Thanks for accepting my opinion.